src/main/kotlin/no/nav/familie/oppdrag/rest/OppdragController.kt

```diff
@@ -32,7 +34,11 @@ class OppdragController(@Autowired val oppdragService: OppdragService,
             oppdragService.opprettOppdrag(utbetalingsoppdrag, oppdrag, 0)
         }.fold(
                 onFailure = {
-                    illegalState("Klarte ikke sende oppdrag for saksnr ${utbetalingsoppdrag.saksnummer}", it)
+                    if (it is OppdragAlleredeSendtException) {
+                        conflict("Oppdrag er allerede sendt for saksnr ${utbetalingsoppdrag.saksnummer}.")
```

Comment:
Dette kommer bli catchet i ApiExceptionHandler og returnere 500?

Reply:
.. I utgangspunktet så er dette error og tasken skal feile, men tanken er at vi gjennom toggle kan error for utvalgte behandlinger ... derfor syns jeg at error er riktig ..Er du enig? ...

Reply:
Ja, men det er ju ikke en error i denne appen. Det er greit hvis konsumenten av tjenesten logger error hvis de føler behov for det. Error trigger alarmer, og jeg synes ikke duplicate er error, men forventet oppførsel

Reply:
... vi ønsker vell at det sendes et alarm som da gjør oss oppmerksomme på at det feiler og da har vi mulighet å bypasse feilet med toggle ... Disse feilen forventes uansett ikke skje så ofte (i så fall må vi ha en annen løsning) ... Det er kanskje enklere om vi diskuterer dette muntlig.

Reply:
Ok, endrer til å logge warning.